### PR TITLE
feat: eslintignore icon

### PR DESCRIPTION
**Pull request by BBboy01**
Same with vscode icons, `.eslintignore` file icon as same as `.eslintrc`

### CODE REVIEW

File: lua/nvim-web-devicons/icons-light.lua [hunk 1]
```diff
@@ -47,6 +47,12 @@ local icons_by_filename = {
     cterm_color = "56",
     name = "Eslintrc",
   },
+  [".eslintignore"] = {
+    icon = "",
+    color = "#4b32c3",
+    cterm_color = "56",
+    name = "Eslintrc",
```
Comment (alex-courtis) [inline lua/nvim-web-devicons/icons-light.lua:54]: Let's make this one unique, maybe

```suggestion
    name = "EslintIgnore",
```

**Review by alex-courtis — CHANGES_REQUESTED** [review]
Many thanks for your contribution... just needs a unique name.

File: lua/nvim-web-devicons/icons-light.lua (hunk 1 above)
Comment (BBboy01) [reply lua/nvim-web-devicons/icons-light.lua]: yeah, you are right

**Review by alex-courtis — APPROVED** [review]
Many thanks for your contribution!

**Pull request merged**